.github/workflows/ci.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
         echo "The old one will disappear after 7 days."
 
     - name: Run bazel build
-      run: bazel build '//...'
+      run: bazel test --test_output=errors //...
```

Comment:
Why do we change this line? Intentionally?

Reply:
Oops - that line was meant to replace line 48 (`bazel test`) so that it will print out the errors if the test fails. Originally, if a test fails, it's hard to debug since you cannot see the errors. Updated this file to keep line 45 the same, and updated line 48.